addons/website/static/src/snippets/s_rate/000.scss

```diff
@@ -0,0 +1,6 @@
+
+.s_features_rate {
+    .s_rate_dish_price {
+        font-size: 18px;
```

Comment:
To control the font-size I would use the h5 class (which is 18px by default I think anyway) on the element. This way no need to support a class or add css.

We have lots of work to improve the way users control font-sizes though.